app-emulation/firecracker-bin/firecracker-bin-0.25.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info
+
+DESCRIPTION="Secure and fast microVMs for serverless computing (static build)"
+HOMEPAGE="https://firecracker-microvm.github.io https://github.com/firecracker-microvm/firecracker"
+SRC_URI="
+	amd64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-x86_64.tgz
+	)
+	arm64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-aarch64.tgz
+	)"
+
+LICENSE="|| ( Apache-2.0 MIT Apache-2.0-with-LLVM-exceptions ) MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RESTRICT="test strip"
+
+RDEPEND="!app-emulation/firecracker
+	acct-group/kvm"
+
+QA_PREBUILT="/usr/bin/firecracker
+	/usr/bin/jailer"
```

Comment:
It's trivial so I'll add it on merge -- missing the new `usr/bin/seccompiler-bin`:
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/seccompiler-bin
```
On a side-note, it's not /essential/ but QA_* variables should use relative paths (aka `="usr/..."` rather than `="/usr/..."` as (briefly and easy to miss) mentioned in the docs:
https://devmanual.gentoo.org/eclass-reference/ebuild/index.html#lbAG

Not that it's necessary to change them all, but since touching this I'll adjust for the latest ebuild.

Reply:
Thanks, I missed that.